src/test/java/org/craftercms/deployer/impl/TargetServiceImplTest.java

```diff
@@ -251,7 +253,7 @@ public void testDuplicateNonExistingTarget() throws TargetServiceException {
         TargetService targetServiceSpy = Mockito.spy(targetService);
         doReturn(false).when(targetServiceSpy).targetExists(ENVIRONMENT, NON_EXISTING_SITE);
         assertThrows(TargetNotFoundException.class,
-                () -> targetServiceSpy.duplicateTarget(ENVIRONMENT, NON_EXISTING_SITE, NEW_SITE_NAME));
+                () -> targetServiceSpy.duplicateTarget(ENVIRONMENT, NON_EXISTING_SITE, NEW_SITE_NAME, false, "test", emptyMap()));
```

Comment:
Should we unit test the new functionality?

Reply:
Added unit tests that validate the parameters are passed to the template on duplicate.